src/tools/showprogress.m

```diff
@@ -0,0 +1,33 @@
+function fout = showprogress(x, whichbar, varargin)
+% showprogress shows waitbars
+%
+% Inputs:
+%   x: percentage in integer (e.g.: 1 = 1%, 40 = 40%, etc.)
+%   whichbar: caption
+%   varagin: see waitbar header for explanation
+% 
+% Output:
+%   fout: handle output from waitbar() (WAITBAR_TYPE = 1)
+%
+% .. Author: 
+%        - Lemmer El Assal (Feb 2017)
+%
+    global WAITBAR_TYPE;
+    fout = [];
+    if ~isempty(WAITBAR_TYPE)
+        switch WAITBAR_TYPE
```

Comment:
@syarra , Do we even need to have the option of a standard waitbar? Would the text waitbar not be sufficient?  I believe so, as I don't see users switching back manually.